html/changelogs/Sindorman-bugfixes.yml

```diff
@@ -0,0 +1,6 @@
+author: PoZe
+
+delete-after: True
+
+changes: 
+  - bugfix: "Wizard disable technology no longer affects caster(wizard), making wizard IPC not take damage. It does still damage other wizard IPC, who is not caster."
```

Comment:
This changelog is a little spaghetti. I would just keep it as "Wizard disable technology spell no longer affects the caster."